VespaGAS/Managers/OilMixCalculator.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+typealias GasType = Float
+typealias OilType = Float
+typealias OilMixType = Float
+
+let MLInLiter: Float = 1000
+
+struct OilMixCalculator {
+    static func oilMLForGasoline(gasInLiter: GasType, oilPercentage: OilMixType) -> OilType? {
+        if oilPercentage == 0 {
+            return nil
+        }
+
+        return gasInLiter * MLInLiter / 100 * oilPercentage
```

Comment:
furthermore, whilst you've used typealiases this is still a classic case of connascence of meaning :) you must know that `GasType` is measure in litres and `OilType` is returned in litres in order for this to be correct. 

I'd recommend creating a `Volume` struct which encapsulates this better. Furthermore, this struct could have behaviour on it. Typically if you have a static function acting on a value object, then that could be a method on the value object itself. 

e.g. compare

`Calculator.multiply(someDomainObject, 4)`

to

`someDomainObject.multiplyBy(4)`